environments/openstack/README.md

```diff
@@ -1,17 +1,135 @@
-# Provisioning with Openstack
+# Deployment on OpenStack
 
-## How to run it ?
+In this tutorial you will learn how to set up MedTagger on OpenStack
+using Terraform and Ansible. 
 
-Download [`Openstack RC file`](https://docs.openstack.org/zh_CN/user-guide/common/cli-set-environment-variables-using-openstack-rc.html) from your Openstack instance. It holds all needed environmental variables.
+### Generating SSH keys
+
+Proceed to your OpenStack Instance and go to **Computations - Access and Security - Key pairs** tab.
+
+Click on **Create Key Pair** button and create 2 sets of key pairs.
+
+**Important**: By default our configurations scripts assume that the keys are named `app-key`
+and `db-key`. If you want to use different names, please change them in `main.tf` and 
+`group_vars/all.yml` files.
+
+The download of the private keys should start automatically after creating them. If not,
+download private keys manually and place then in **Ansible directory.**
```

Comment:
What is "Ansible directory"?

Reply:
Yeaaa :smile: I'll change it to full path to the folder - environments/openstack/ansible